From 903dd37fa53cb166f12c97a077e15b1e2fde4251 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20Br=C3=BCns?= <stefan.bruens@rwth-aachen.de>
Date: Mon, 18 Nov 2019 19:07:46 +0100
Subject: [PATCH 1/3] Fix build with gtk-doc >= 1.32 due to non-existing
 tree_index.sgml

The tree_index.sgml is no longer generated when the library does not
provide any GObjects.

See https://gitlab.gnome.org/GNOME/gtk-doc/issues/103
---
 docs/libdbusmenu-glib/reference/libdbusmenu-glib-docs.sgml | 4 ----
 docs/libdbusmenu-gtk/reference/libdbusmenu-gtk-docs.sgml   | 4 ----
 2 files changed, 8 deletions(-)

diff --git a/docs/libdbusmenu-glib/reference/libdbusmenu-glib-docs.sgml b/docs/libdbusmenu-glib/reference/libdbusmenu-glib-docs.sgml
index 76258e5..3909024 100644
--- a/docs/libdbusmenu-glib/reference/libdbusmenu-glib-docs.sgml
+++ b/docs/libdbusmenu-glib/reference/libdbusmenu-glib-docs.sgml
@@ -18,10 +18,6 @@
     <xi:include href="xml/types.xml"/>
 
   </chapter>
-  <chapter id="object-tree">
-    <title>Object Hierarchy</title>
-     <xi:include href="xml/tree_index.sgml"/>
-  </chapter>
   <index id="api-index-full">
     <title>API Index</title>
     <xi:include href="xml/api-index-full.xml"><xi:fallback /></xi:include>
diff --git a/docs/libdbusmenu-gtk/reference/libdbusmenu-gtk-docs.sgml b/docs/libdbusmenu-gtk/reference/libdbusmenu-gtk-docs.sgml
index e2e3c9f..9219ea5 100644
--- a/docs/libdbusmenu-gtk/reference/libdbusmenu-gtk-docs.sgml
+++ b/docs/libdbusmenu-gtk/reference/libdbusmenu-gtk-docs.sgml
@@ -17,10 +17,6 @@
     <xi:include href="xml/parser.xml"/>
 
   </chapter>
-  <chapter id="object-tree">
-    <title>Object Hierarchy</title>
-     <xi:include href="xml/tree_index.sgml"/>
-  </chapter>
   <index id="api-index-full">
     <title>API Index</title>
     <xi:include href="xml/api-index-full.xml"><xi:fallback /></xi:include>
-- 
2.24.0

